WORKSPACE

```diff
@@ -3,8 +3,27 @@ workspace(name = "com_github_zaucy_protoc_gen_angular")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 http_archive(
-  name = "com_google_protobuf",
+  name = "com_google_protobuf_old",
```

Comment:
not sure if this will work. need to test with the gateway

Reply:
I switched to the gateway version of protobuf, should it work now @ken31ee ?

Reply:
Theoretically yes. The app and gateway should be able to comunicate without problem